stdlib/public/CTensorFlow/ctensorflow_init.cpp

```diff
@@ -17,33 +17,6 @@ void handle_sigint(int signal) {
   exit(1);
 }
 
-void InitTensorFlowRuntime(unsigned char enable_debug_logging,
-                           int verbose_level) {
-  // Install a signal handler to ensure we exit when interrupted.
-  signal(SIGINT, handle_sigint);
```

Comment:
@eaplatanios, I noticed that the Swift implementation does not have this, which was need to ensure `^C` kills the program. Is it easy to add this to the swift implementation?

Reply:
Good catch. I added this and will soon make a PR in swift-apis.